.github/workflows/template_build.yml

```diff
@@ -30,6 +30,7 @@ on:
 
 env:
   TOY_SECRET_GITHUB: ${{ secrets.TOY_SECRET }}
+  GHP_TOKEN: ${{ secrets.GHP_TOKEN }}
```

Comment:
So we actually already check out the product-metadata repo in the steps below. I'm wondering if it makes sense to not invoke the Github repo clone python function. In general, this type of thing feels like environment setup, especially in the case that you're developing locally. E.g. if you want to build template DB with modified product-metadata on your machine. So I'd prefer we rely on cloning in the GHA, and not in the export step. Not a blocker though. (can discuss later)

Reply:
yea this was tough to place. I went with python approach since having the org's metadata feels like a requirement of using these new parts of `dcpy`, so it felt weird to not have the approach to getting org metadata in `dcpy`

definitely feels like environment setup, just gotta figure out (later) what `dcpy` shouldn't be able to do

Reply:
Definitely agree on the awkwardness of it. We can brainstorm some ways to make it more standard, e.g. retrieving it could be a part of dcpy setup/install. The reason I still favor just cloning it in the GHA (or in this case, using what's already cloned in this GHA job) retrieving the metadata repo feels more akin to injecting passwords into the environment, or installing dependencies, basically things that should be done at the beginning of the job, rather than tucked in a python script near the end. (cloning in GHA also mirrors our local dev setup.)